client/assets/js/main.js

```diff
@@ -21,4 +21,54 @@ const shortenURL = () => {
       document.getElementById('shortened-url-lists').innerHTML = data.message
   })
 }
+
 document.querySelector('.create-shorten-url').onclick = shortenURL
+
+renderurls(urls)
+
+//===================================
+const verifyURLs = (e) => {
+	if(e.target.elements.shortenedURL.value !== '' && e.target.elements.shortenedURL.value.length < 7){
+		console.log('Too short.')
+		return 1 //1 means the short URL is too short.
+	} else if(e.target.elements.originalURL.value.length >= 0) {
+		return fetch(e.target.elements.originalURL.value, { mode: 'no-cors' })
+			.then(response => {
+				console.log('Success! I think.')
+				return 0 //0 means everything is fine.
+			})
+			.catch(error => {
+				console.log('That URL does not exist.')
```

Comment:
I think this could be misleading. A 404 could be caught, but the URL does indeed exist (what if I want to share a cool 404 page with my friends?)